.github/workflows/test_asset-transfer.yml

```diff
@@ -556,6 +610,13 @@ jobs:
           ./bin/fabric-cli configure network --local-network=network2
           ./scripts/initAssetsForTransfer.sh
         working-directory: samples/fabric/fabric-cli
+                
+      - name: Fabric Sync Membership using IIN Agent
+        run: |
+          ./bin/fabric-cli configure membership --local-network=network1 --target-network=network2 --iin-agent-endpoint=localhost:9500
+          ./bin/fabric-cli configure membership --local-network=network2 --target-network=network1 --iin-agent-endpoint=localhost:9501
+          sleep 10
```

Comment:
Is 10 enough to sync?

Reply:
It did pretty fast in my VM (within a sec or two), may be because there's only one node on each side. So I thought 10 should be sufficient.